### ISSUE TITLE
Poor handling of SteamVR crash

### ISSUE THREAD

**Issue by ubershy** [alvr-org/ALVR#562]
### Bug report:

TLDR: If SteamVR crashes for whatever reason, it is impossible to restart SteamVR and ALVR while wearing headset. Also there are orphaned Chrome processes made by ALVR.

#### How to reproduce

1. Start ALVR. SteamVR will automatically start.
2. Wait for SteamVR to crash (it usually happens once in two hours for me). Or just kill "VR Server" (vrserver.exe) via task manager. In windows task manager it is "End task" option from context menu.

##### Bad things which happen next:
1. On PC SteamVR will show window "SteamVR has encountered critical error" with options to restart SteamVR or quit SteamVR. No matter which option you choose, other bad things listed below still happen.
2. It would be impossible to restart SteamVR via "Restart SteamVR" button in ALVR window on PC. The button does nothing. ALVR web interface in Oculus Browser becomes inacessible too.
3. Because of things mentioned above it is impossible to make ALVR and SteamVR running properly again while wearing headset. The only way to do it is to put headset off, go to PC, close SteamVR, close ALVR, start ALVR again. For me it is very inconvenient because my PC is located in another room on another floor.
4. There also will be orphaned tree of Chrome processes (started by ALVR) that can only be killed manually via task manager or via PC restart. They will not be killed on closing ALVR or restarting SteamVR, etc. These trees can accumulate with multiple crashes.

Here is a screenshot of process hacker task manager showing a tree of ALVR chrome processes with parent "vrserver.exe" before SteamVR crash:
![Annotation 2021-03-24 213048](https://user-images.githubusercontent.com/11366070/112370524-91229580-8cee-11eb-8157-6284bb986241.png)
After SteamVR crash they are orphaned:
![Annotation 2021-03-24 213218](https://user-images.githubusercontent.com/11366070/112371239-6f75de00-8cef-11eb-9ef6-ef88d94fccee.png)

#### Environement

> What hardware do you have: CPU, GPU, Audio

Ryzen 2700x, GTX 1070, ur28m sound card

> What version of ALVR and SteamVR have you tried. 

ALVR 15.1.0, SteamVR 1.16.10

> Do you have the latest Windows updates? Run "winver" in Windows 10 to get the version number

Windows 10, Version 2004

**Comment by thegforcian** [comment]
I'm having this exact problem as a loop in which the ALVR addon is now permanently blocked by SteamVR since it wants to restart every time I unblock the application and it reverts into a blocked state.  Attached are 7 crash timestamps from my config directory.  Please let me know if I should add a new issue but this seems so similar I decided to simply comment.
[vrserver_crash_timestamp1.txt](https://github.com/alvr-org/ALVR/files/6209350/vrserver_crash_timestamp1.txt)
[vrserver_crash_timestamp2.txt](https://github.com/alvr-org/ALVR/files/6209351/vrserver_crash_timestamp2.txt)
[vrserver_crash_timestamp3.txt](https://github.com/alvr-org/ALVR/files/6209352/vrserver_crash_timestamp3.txt)
[vrserver_crash_timestamp4.txt](https://github.com/alvr-org/ALVR/files/6209354/vrserver_crash_timestamp4.txt)
[vrserver_crash_timestamp5.txt](https://github.com/alvr-org/ALVR/files/6209355/vrserver_crash_timestamp5.txt)
[vrserver_crash_timestamp6.txt](https://github.com/alvr-org/ALVR/files/6209356/vrserver_crash_timestamp6.txt)
[vrserver_crash_timestamp7.txt](https://github.com/alvr-org/ALVR/files/6209357/vrserver_crash_timestamp7.txt)

**Comment by ubershy** [comment]
@thegforcian 
I think your problem deserves a separate issue on github.

But you have reminded me of SteamVR blocking ALVR addon which I forgot to mention in this issue. It is very inconvenient when it happens.
My pc is far from the play area. I managed to put ALVR restart script to a shortcut on my wireless keyboard and it helps. But when SteamVR blocks ALVR addon after crash, there is no way I can reenable it remotely with some script. I have to go to PC to resolve it manually.

**Comment by ronthecookie** [comment]
The crash doesn't necessarily have to be easy to recover from; finding the source of these random crashes could be more productive.

**Comment by ubershy** [comment]
@ronthecookie 
Yes, you're right. Also the users would be more willing to report crashes especially if crashes are hard to deal with...

**Comment by Yang-PM** [comment]
There are some major crashes in steamVR, sometimes when Quest connects to the host, sometimes when the game starts, I don't know how to check the cause of this crash.

**Comment by ubershy** [comment]
> There are some major crashes in steamVR, sometimes when Quest connects to the host, sometimes when the game starts, I don't know how to check the cause of this crash.

I experience a lot of crashes as well. I wonder how can I see the cause of client crashes. And how can I see the cause of server crashes, without relying on analyzing SteamVR crash dumps in windbg64, or possibly setting up development environment for ALVR and running it in debug mode.

**Comment by mrstuff** [comment]
same same here happy to reproduce and post logs , wish i knew how to help or what to do to offer useful error reporting, 
 ill see if i cant find some general debug program ,  what bout the one in the oculus dev hub? thats only the quest side tho right? 
my bad if this is in the wrong spot and hey thanks a bunch ya'll
